packages/office-ui-fabric-react/src/components/Button/examples/fluent/compose.tsx

```diff
@@ -4,6 +4,7 @@ import { mergeCss } from '@uifabric/merge-styles';
 
 // TODO:
 // 1. how do we know the slots for component?
+//    Why do we need to?
```

Comment:
good question. i think we can just merge the keys in theme `styles` and `variants` to get all the `slotNames`? @mnajdova

Reply:
talked to marjia, yeah we dont need #1. i can fix this once you merge your change :)